inc/commondbtm.class.php

```diff
@@ -3062,7 +3062,7 @@ function getLinkedItemsCount() {
    function getField($field) {
 
       if (array_key_exists($field, $this->fields)) {
-         return $this->fields[$field];
+         return $this->fields[$field] ?? "";
```

Comment:
`getField` is used at many places, it may break strict tests (like `$item->getField('xxx') === null`).
I know this method should be used only for display, but I am not sure everyone is aware of that.

Reply:
That what Alex was saying: using historical getters is strongly discouraged in GLPI because that often has a... say particular way of working.

Reply:
After struggling 1h~ with the tests (and not find the issue), i revert this part